torchbenchmark/util/triton_op.py

```diff
@@ -817,7 +817,7 @@ def ncu_trace(self, input_id: int, fn_name: str, replay: bool=False) -> str:
         import subprocess
         from pathlib import Path
 
-        op_task_args = copy.deepcopy(sys.argv)
+        op_task_args = [sys.executable] + copy.deepcopy(sys.argv)
```

Comment:
This might not work for fbcode command. Can you please test this using `buck run`?